modal/requirements.txt

```diff
@@ -1,5 +1,6 @@
 # Pins Modal dependencies installed within the container runtime.
-aiohttp==3.8.3
+aiohttp==3.8.3;python_version<'3.12'
+aiohttp==3.9.1;python_version>='3.12'
```

Comment:
Is this going to force a rebuild of all user images?

Reply:
Yeah, it will

Reply:
We have not changed requirements.txt for a long time, since before we were concerned about rebuilds. Maybe we'll need to figure that out next 😬

Reply:
Hum, I wonder if we could split out the base requirements file by Python version 🤔

Reply:
OK gave that a shot